tests/app/validation/test_number_range_validator.py

```diff
@@ -110,24 +110,25 @@ def test_within_range_at_max(self):
             self.fail("Valid integer raised ValidationError")
 
     def test_manual_min(self):
-        answer = { 'min_value': {
-                        "value": 10
-                    },
-                   'label': 'Min Test',
-                   'guidance': '',
-                   'options': [],
-                   'mandatory': False,
-                   'validation':
-                      {'messages':
-                           {
-                            'INVALID_NUMBER': 'Please only enter whole numbers into the field.',
-                            'NUMBER_TOO_SMALL': 'The minimum value allowed is 10. Please correct your answer.'
-                           }
-                      },
-                   'description': '',
-                   'id': 'test-range',
-                   'type': 'Currency'
-                  }
+        answer = {
+            'min_value': {
+                "value": 10
+            },
+            'label': 'Min Test',
+            'guidance': '',
+            'options': [],
```

Comment:
only because you touched it, we shouldn't need to specify the values that we don't need.